### PR TITLE
Added new logos

### PR DESCRIPTION
**Pull request by RoundDev**
@amberolive @zachnthebox @mhvolk @fdlane @KeithClinard @donaldnash1989 
Updated:

- New team logo for forgot password form

- New team logo for navbar

- Added new formatting in CSS for nav-brand and new class for img-brand

### CODE REVIEW

File: app/styles/app.css [hunk 1]
```diff
@@ -29,6 +29,19 @@ over-ride "Weak" message, show font in dark grey
         padding: 16px 15px;
```
Comment (RoundDev) [inline app/styles/app.css]: the padding needs to be changed  to 0px 15px

File: app/styles/app.css [hunk 2]
```diff
@@ -29,6 +29,19 @@ over-ride "Weak" message, show font in dark grey
...
+
+    }
+
+    .navbar-brand {
+      padding: 0px 0px;
+      float: left;
+    }
+
+    .img-brand {
+      padding: 16px 15px;
```
Comment (RoundDev) [inline app/styles/app.css]: padding needs tp be changed to 0px 0px

File: app/styles/app.css (hunk 1 above)
Comment (MaudeVolk) [inline app/styles/app.css]: Just correct the spelling of administrator on line 20.

**Comment by MaudeVolk** [comment]
Keep up the good work.

**Pull request closed without merging**